src/component/user/Login/Login.tsx

```diff
@@ -7,17 +7,18 @@ import { DEMO_TYPE } from '../../../constants/authTypes';
 import Authentication from '../Authentication/Authentication';
 import { useAuthDetails } from '../../../hooks/api/getters/useAuth/useAuthDetails';
 import { useAuthUser } from '../../../hooks/api/getters/useAuth/useAuthUser';
-import { Redirect } from 'react-router-dom';
+import { Redirect, useHistory } from 'react-router-dom';
 
 const Login = () => {
     const styles = useStyles();
     const { authDetails } = useAuthDetails();
     const { user } = useAuthUser();
     const query = useQueryParams();
     const resetPassword = query.get('reset') === 'true';
+    const redirect = query.get('redirect') || '/';
 
     if (user) {
-        return <Redirect to="/features" />;
```

Comment:
I suppose we should still redirect to the features page on login. Does that happen somewhere else?

Reply:
Yes. so if there's no redirect we will get the `/` and this is matched in Apps.ts L:85
```
<ProtectedRoute
    exact
    path="/"
    unauthorized={isUnauthorized()}
    component={Redirect}
    renderProps={{ to: '/features' }}
/>
```